SebastianHerrera/bootcamp-java/src/main/java/hackerrank/day3/arraylist/ExerciseArraylist.java

```diff
@@ -0,0 +1,43 @@
+package hackerrank.day3.arraylist;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * @author Sebastian Herrera (bermeosebas@gmail.com)
+ * @created 31/08/21
+ */
+public class ExerciseArraylist {
+
+    public static final String ERROR_MESSAGE = "ERROR!";
+
+    public static void getArrayInfo() {
+
+        Scanner scanner = new Scanner(System.in);
+
+        int numberLines = scanner.nextInt();
+
+        ArrayList<ArrayList<Integer>> dynamicTable = new ArrayList<>();
```

Comment:
Is completely neccesary this "inception" ? ArrayList<ArrayList< ??
Also remember Code for Interface, not for Implementation:
https://www.javaguides.net/2018/06/java-collection-framework-best-practices.html

Reply:
Oh, okay. I think I have another solution